cmd/images-back/main.go

```diff
@@ -43,6 +46,25 @@ func main() {
 	}
 	defer dbPsql.Close()
 
+	//dot, err := dotsql.LoadFromFile("migrations/20190909154444_image_files_table.up.sql")
+	//_, err = dot.Exec(dbPsql, "create-users-table")
+
+	driver, err := postgres.WithInstance(dbPsql, &postgres.Config{})
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer driver.Close()
+	m, err := migrate.NewWithDatabaseInstance(
+		"file://migrations",
+		"postgres", driver)
+	if err != nil {
+		log.Fatal(err)
+	}
+	err = m.Up()
+	if err != nil {
```

Comment:
Let's add an additional test for ErrNoChange error. It is an error that raises up when there are no change is required. Right now, If you have no new migrations, the service will not start. 

```suggestion
	if err != nil && err != migrate.ErrNoChange{
```